lib/index.js

```diff
@@ -140,19 +140,14 @@ function getOptions(options, cb) {
   options.style = getStyle(options) || 0;
 
   // context object represents node-sass environment
-  options.context = { options: options };
-
-  var error = options.error;
-  var success = options.success;
+  options.context = { options: options, callback: cb };
```

Comment:
Would this not overwrite the one your are setting in render.js? `renderOptions.context = { .. };`